roles/install/vars/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+__default_pkgs:
+  - lemonldap-ng
+  - lemonldap-ng-doc
+
+__redhat_apache_pkgs:
+  - httpd
+  - mod_perl
+  - mod_fcgid
+__redhat_nginx_pkgs:
+  - nginx
+  - lemonldap-ng-fastcgi-server
+  - lemonldap-ng-nginx
+__redhat_default_pkgs:
+  - perl-LWP-Protocol-https
```

Comment:
This is missing in the RPMs Requires, I will fix that, thanks :-)

Reply:
so let me know when it's done on the RPM side and i will also remove that package

Reply:
The dependency has been explicitly added with https://gitlab.ow2.org/lemonldap-ng/lemonldap-ng/-/merge_requests/513

But anyway, this was already an indirect dependency from perl-Soap-Lite and perl-Sentry-Raven.

Reply:
so i removed that part!